src/PIL/Image.py

```diff
@@ -1821,6 +1821,39 @@ def resize(self, size, resample=NEAREST, box=None):
 
         return self._new(self.im.resize(size, resample, box))
 
+    def reduce(self, factor, box=None):
```

Comment:
The name of the operation and argument could be discussed.

Reply:
@python-pillow/pillow-team any objections to naming?

Reply:
Well, I can't think of a better name :)

Reply:
thumbs up for the current naming!